src/content/advanced/cost-optimization/workload-clusters-scale-down/index.md

```diff
@@ -0,0 +1,199 @@
+---
+linkTitle: Scaling down
+title: Workload Clusters Scale Down
+description: In order to save costs over the weekends or stale periods some entities would like to scale down the clusters to 0 nodes, this guide describes such process for Giant Swarm clusters.
+weight: 100
+menu:
+  main:
+    parent: cost-optimization
+last_review_date: 2021-11-15
+user_questions:
+- How can I scale down cluster nodes to 0?
+- How can I silence alerts from the clusters?
+aliases:
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+---
+
+# Periodically scaling down workload clusters to 0 worker nodes
+
+For limiting costs of workload clusters that are used periodically or have stale periods of time, there are use cases to scale down the worker nodes to zero. Such scenarios apply often for development clusters that are not being used over the weekends and there is limited possibility to scale down workloads completely.
+Moreover in order to maximize fully the cost optimization, there should be no workers running at all.  
+
+However as Giant Swarm is also running several components on the workload clusters that are being monitored, this can result in our Giant Swarm Oncall to be paged due to unavailability of such services.
+The following tutorial describes a way to silence periodically alerts from specified clusters and scale down the worker nodes accordingly.
+
+## Silences
+
+Internally Giant Swarm introduced a concept of Silence Custom Resources. They are utilized to silence either particular alerts or whole clusters in case of known persistent issues or alerts that can be omitted for a while.
+The resource consists of constraints and filters which define the silencing rules on defined alerts and clusters. Those are placed in a private github repository and are synced with every Management Cluster via [silence-operator](https://github.com/giantswarm/silence-operator).
+
+You can see the example of such Custom Resource below:
+
+```yaml
+apiVersion: monitoring.giantswarm.io/v1alpha1
+kind: Silence
+metadata:
+  name: INSTALLATION-CLUSTER-ID-silence-cluster
+spec:
+  targetTags:
+  - name: installation
+    value: YOUR_INSTALLATION_NAME
+  matchers:
+  - name: cluster_id
+    value: CLUSTER_ID
+    isRegex: false
+```
```

Comment:
In the Management API, there is no such thing as a cluster ID. And in the new incarnation of the UIs there isn't either. What used to be called the cluster ID is now the cluster name.

When in doubt, explain that the cluster name to use here is the name of the `Cluster` resource and the unique identifier of the cluster.